src/interfaces/special-resolution-interfaces/signing-person-interface.ts

```diff
@@ -0,0 +1,12 @@
+export interface SigningPersonIF {
+  givenName: string
+  additionalName?: string
+  familyName: string
+  email?: string
+}
+
+export const EmptySigningPersonIF: SigningPersonIF = {
+  givenName: '',
+  familyName: '',
+  additionalName: null
+}
```

Comment:
Additional Name is optional but null.
Email is also optional. Should it be initialized here?

Reply:
Additional Name if we didn't define null have to put additional check html
we don't have an email in design

Reply:
I'm looking for consistency between the interface and this empty object. Why are additional name and email different? I'm thinking they should be declared and initialized the same...?

Reply:
my bad, email is not there removed from interface

Reply:
Bye bye, email.